values-it/strings.xml

```diff
@@ -402,54 +403,54 @@
 \n
 \nÈ permesso un solo programma radio per volta. I programmi vecchi saranno rimossi.</string>
     <string name="draft_lighthouse00_title">Faro</string>
-    <string name="draft_lighthouse00_text">Non c\'è nessuna barca ma c\'è un faro .... strano.</string>
-    <string name="draft_road02_title">Autostrada</string>
+    <string name="draft_lighthouse00_text">Non c\'è nessuna barca ma c\'è un faro... strano.</string>
+    <string name="draft_road02_title">Superstrada</string>
     <string name="draft_road02_text">Connessione estremamente rapida tra due luoghi.</string>
     <string name="budget_road">Strade</string>
-    <string name="topic_birthday_phoenix">Buon compleanno phoenix!</string>
+    <string name="topic_birthday_phoenix">Buon compleanno Phoenix!</string>
```

Comment:
same goes for pheonix

Reply:
Oh I see, thanks for letting me notice, I just considered them as names, sorry. You can leave those not capitalized then, or I can edit it, as you wish. 
Anyway, let me know if the other strings are ok.